setup.nsi

```diff
@@ -183,9 +183,9 @@ Section "Required Files"
     File "target\${PROJECT_FINAL_NAME}.exe"
     File "target\${PROJECT_FINAL_NAME}-uber.jar"
     File "src\main\resources\img\icons\dataloader.ico"
+    File "/oname=defaultConfig.properties" "target\config.properties"
```

Comment:
Same as Mac profile above, for Windows we are making a default config to be used when user doesn't have a config yet.